packages/@react-aria/interactions/test/useFocusVisible.test.js

```diff
@@ -90,6 +91,97 @@ describe('useFocusVisible', function () {
 
     expect(el.textContent).toBe('example');
   });
+
+  describe('Setups global event listeners in a different window', () => {
```

Comment:
while it is good to test toggling tabs/windows, it'd be good to have a straightforward test without that. don't remove these ones, just an addition

it'd also be good to include an unmount test